src/helpers/utility.js

```diff
@@ -555,6 +569,7 @@ export function getEnvConfidentialAPIURL() {
 }
 // write to audit log
 export function writeToLog(message, level, params) {
+  // console.log("log params ", params, " message ", message);
```

Comment:
might be a good idea to write everything to browser console log first, but as-is this isn't doing anything :)
```suggestion
```